src/app/component/ReviewStar/ReviewStar.style.scss

```diff
@@ -0,0 +1,43 @@
+.ReviewStar {
+    --star-outline-color: #{$dusty-grey};
+
+    // reset from input default styles
+    display: inline;
+    margin: 0;
+    padding: 0;
+    background: none;
+    border: 0;
+
+    transition: color 150ms ease-in;
+    color: $white;
+    text-shadow:
+        -1px 0 var(--star-outline-color),
+        0 1px var(--star-outline-color),
+        1px 0 var(--star-outline-color),
+        0 -1px var(--star-outline-color);
+
+    font-size: 40px;
+    line-height: 1;
+    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+    font-family: Times; // make sure ★ appears correctly
+    letter-spacing: 3px;
+
+    &::after {
+        content: '★';
+    }
+
+    &:checked,
+    &:hover,
+    &[checked],
+    &:checked ~ input,
+    &:hover ~ input,
+    &[checked] ~ input {
+        --star-outline-color: #{$primary-base-color};
+
+        color: $primary-base-color;
+    }
+
+    &:focus {
+        border: 0;
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4478891)